qiskit/quantum_info/operators/pauli.py

```diff
@@ -10,74 +10,11 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
+# pylint: disable=unused-import
+
 
 """
 DEPRECATED Tools for working with Pauli Operators.
 """
 
-from warnings import warn
-import numpy as np
-from qiskit.exceptions import QiskitError
 from qiskit.quantum_info.operators.symplectic.pauli import Pauli
```

Comment:
Are we leaving `Pauli` here to maintain the `import` path, but it's meant to be deprecated?  We probably want to issue a deprecation warning for this.  We can do that by making the file look like:
```python
def __getattr__(name):
    if name == "Pauli":
        from qiskit.quantum_info.operators.symplectic import Pauli
        warnings.warn(...)
        return Pauli
    raise AttributeError(f"module '{__name__}' has no attribute '{name}'")
```

As an example, see https://github.com/Qiskit/qiskit-terra/blob/ed4e1a397d7faa38274f98c54ba5750043c32929/qiskit/visualization/__init__.py#L150-L161.

Reply:
I've fixed this in 69301eec0.